### PR TITLE
CLOUD-4080 - EAP XP 4.0 Images

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.redhat.com/browse/CLOUD-4080
Signed-off-by: Ken WIlls <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: eap-xp4/runtime-image/test/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Be sure to update eap-testapp with the name of the image generated with s2i.
+FROM jboss-eap-7/eap-xp3-openjdk11-runtime-openshift-rhel8
```
Comment (jmesnil) [inline eap-xp4/runtime-image/test/Dockerfile]: base image is still pointing to EAP XP 3

**Review by jmesnil — CHANGES_REQUESTED** [review]
Outside of my comment, this looks good to me.

That's great to have the JDK17 image built (even though it might not be published when EAP XP 4 is released).

File: eap-xp4/image.yaml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
+                  ref: EAP_XP4_RC1
+          - name: jboss-eap-7-image
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+                  ref: EAP_XP4_RC1
+          - name: wildfly-cekit-modules
+            git:
+                  url: https://github.com/wildfly/wildfly-cekit-modules.git
+                  ref: 0.26.0
```
Comment (jfdenise) [inline eap-xp4/image.yaml]: 0.26.0 is for WF26. We are still on EAP 7.4.x, should be 0.23.4

**Comment by luck3y** [comment]
Ok, cool -- thanks I wanted to make sure with you.

**Comment by luck3y** [comment]
@jmesnil @jfdenise Thanks, updated.

**Comment by luck3y** [comment]
This should be all set now.

**Review by jmesnil — DISMISSED** [review]

**Review by jmesnil — APPROVED** [review]

**Pull request merged**